openconcept/thermal/tests/test_thermal_comps.py

```diff
@@ -3,7 +3,7 @@
 import numpy as np
 from openmdao.utils.assert_utils import assert_near_equal, assert_check_partials
 from openmdao.api import Problem, NewtonSolver, DirectSolver
-import openconcept.components.thermal as thermal
+import openconcept.thermal as thermal
```

Comment:
Is the entire thermal api imported here?  That seems different than the other tests where only the classes that are used are imported.

Reply:
Changed to match others